py/get_tmdb.py

```diff
@@ -29,13 +35,17 @@ def split_movie_data(self, imdb_id, api_data):
         images_data = api_data["images"]["posters"]
         backdrops_data = api_data["images"]["backdrops"]
         similar_movies = api_data["similar"]["results"]
-        alternate_languages = api_data["translations"]["translations"]
+        translations = api_data["translations"]["translations"]
         videos = api_data["videos"]["results"]
-        keywords = api_data["kewords"]["keywords"]
+        keywords = api_data["keywords"]["keywords"]
         lists = api_data["lists"]["results"]
         changes = api_data["changes"]["changes"]
         trailers = api_data["trailers"]["youtube"]
-        release_dates_raw = api_data["release_dates"]["results"]["release_dates"]
+        release_dates = api_data["release_dates"]["results"][0]["release_dates"]
+
+        for dict in cast_data:
```

Comment:
I don't know if using dict like this does something special but using default keywords for variable names are usually not preferable.